src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -217,6 +219,24 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
       }
     }
   }
+  /* Return a Markdown table of all entries in the database */
```

Comment:
Good point. I suspect there'll rapidly be a ticket saying "as jeff, make this toggleable" but I'll wait and see.